README.md

```diff
@@ -67,13 +67,13 @@ docker run -it \
     -p 3000:3000 \
     --add-host host.docker.internal:host-gateway \
     --name openhands-app-$(date +%Y%m%d%H%M%S) \
-    ghcr.io/opendevin/opendevin:0.8
+    ghcr.io/openhands/openhands:0.8
```

Comment:
These should be `all-hands-ai/openhands`

Reply:
Thanks, will update now